bench/comparison/Makefile

```diff
@@ -101,10 +69,10 @@ hosts-lookup:
 	$(node_command) run.js hosts-lookup $(run_options)
 
 deps:
-	yarn download-deps
+	cd ../.. && yarn && yarn build
+	yarn install --immutable
```

Comment:
Maybe we don't need to install but could import relatively from here: `bench/comparison/blockers/cliqz-base.js`?

Reply:
relative packages does not work with yarn as it yarn forces checksum check